code/datums/wires/airlock.dm

```diff
@@ -132,6 +127,12 @@
 			A.lights = !A.lights
 			A.update_icon()
 
+/obj/machinery/door/airlock/proc/reset_ai_wire()
+	if(aiControlDisabled == 1)
+		aiControlDisabled = 0
+	else if(aiControlDisabled == 2)
+		aiControlDisabled = -1
```

Comment:
Do we have defines for these magic numbers?

Reply:
fuck knows